sklearn/metrics/tests/test_pairwise.py

```diff
@@ -1273,3 +1273,27 @@ def test_pairwise_distances_data_derived_params(n_jobs, metric, dist_function,
 
         assert_allclose(dist, expected_dist_explicit_params)
         assert_allclose(dist, expected_dist_default_params)
+
+
+@pytest.mark.parametrize("metric", ["seuclidean"])
```

Comment:
You don't need to parametrize for a single value

Reply:
Sorry, I had originally tried to include 'mahalanobis' in the tests, but it kept failing because one of the Windows test benches didn't like me trying to use `np.double`.  

It was throwing an error  in one of the `np.linalg` libraries, so I removed it as I didn't think it was necessary to test for this purpose.